src/KpmRepoManager.ts

```diff
@@ -1,185 +1,179 @@
 import {
-    wrapExecPromise,
-    isWindows,
-    getRootPaths,
-    normalizeGithubEmail,
-    getFileType
+  wrapExecPromise,
+  isWindows,
+  getRootPaths,
+  normalizeGithubEmail,
+  getFileType,
 } from "./Util";
 
 function getProjectDir(fileName = null) {
-    let projectDirs = getRootPaths();
+  let projectDirs = getRootPaths();
 
-    if (!projectDirs || projectDirs.length === 0) {
-        return null;
+  if (!projectDirs || projectDirs.length === 0) {
+    return null;
+  }
+
+  // VSCode allows having multiple workspaces.
+  // for now we only support using the 1st project directory
+  // in a given set of workspaces if the provided fileName is null.
+  if (projectDirs && projectDirs.length > 0) {
+    if (!fileName) {
+      return projectDirs[0];
     }
 
-    // VSCode allows having multiple workspaces.
-    // for now we only support using the 1st project directory
-    // in a given set of workspaces if the provided fileName is null.
-    if (projectDirs && projectDirs.length > 0) {
-        if (!fileName) {
-            return projectDirs[0];
-        }
-
-        for (let i = 0; i < projectDirs.length; i++) {
-            const dir = projectDirs[i];
-            if (fileName.includes(dir)) {
-                return dir;
-            }
-        }
+    for (let i = 0; i < projectDirs.length; i++) {
+      const dir = projectDirs[i];
+      if (fileName.includes(dir)) {
+        return dir;
+      }
     }
-    return null;
+  }
+  return null;
 }
 
 async function getCommandResult(cmd, projectDir) {
-    let result = await wrapExecPromise(cmd, projectDir);
-    if (!result) {
-        // something went wrong, but don't try to parse a null or undefined str
-        return null;
-    }
-    result = result.trim();
-    let resultList = result
-        .replace(/\r\n/g, "\r")
-        .replace(/\n/g, "\r")
-        .replace(/^\s+/g, " ")
-        .replace(/</g, "")
-        .replace(/>/g, "")
-        .split(/\r/);
-    return resultList;
+  let result = await wrapExecPromise(cmd, projectDir);
+  if (!result) {
+    // something went wrong, but don't try to parse a null or undefined str
+    return null;
+  }
+  result = result.trim();
+  let resultList = result
```

Comment:
can you leave a comment on what this should look like before and after the replace?

Reply:
will do

Reply:
moved this and other command related to the util and added comments.